docs/tests_in_tilt.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Tests in Tilt
+description: "Run your tests from Tilt for visibility, responsiveness, and fine-grained control."
+layout: docs
+---
+With the Tilt `test` primitive, you can run your tests from Tilt itself. Take advantage of Tilt's file-watching and responsiveness, and the power and customizability of Starlark, to supercharge your testing workflow.
```

Comment:
I'd say for the last bit, something like "...to include tests with the rest of your workflow in Tilt."
Just to be more modest and descriptive. What do you think?

Reply:
+1 for modesty